.github/workflows/r.yml

```diff
@@ -11,7 +11,7 @@ on:
     - cron: "0 13 * * *"
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
```

Comment:
This workflow is a good example. `push` and `schedule` trigger this workflow with `github.ref=master`. If `push` triggers a new run while a run triggered by `schedule`, the schedule run will be canceled. This PR prevents it.

Reply:
<img width="884" alt="image" src="https://github.com/mlflow/mlflow/assets/17039389/2af45ea5-966a-4664-b7fa-c1efa6c40cbc">

https://github.com/mlflow/mlflow/actions/runs/5267477791